### DISCUSSION TITLE
horizontal Swiper - add shadow to slides

### DISCUSSION THREAD

**Discussion by rouaida** [nolimits4web/swiper#4618] [Q&A]
I need to apply a shadow on the slides but it is always cut on both sides.

## Actual Behavior

if I add padding I would have an alignment problem with the title above
if I work on the width of the container to adjust the alignment I would have a swiping problem: instead of the element disappearing at the title level it disappears a little more to the left where the padding is present for the conservation of the shadow

## Expected Behavior

Is there a way to respect the following points at the same time:
1) having the shadow on both sides
2) align the slides with the block title
3) when swiping the slide disappears at the title level not more to the left

Thanks,

**Comment by nolimits4web** [comment]
padding + negative margin should do the trick https://codesandbox.io/s/swiper-slides-per-view-forked-wg490

**Reply by rouaida** [reply]
Yes this is an option
thank you for your reply :)
But in this case num 3 is not respected: while swipping the container overflow is not aligned to the parag above

**Comment by rouaida** [comment]
Any suggestions please? or can't be done : applying the 3 points?
